src/main/java/org/shanerx/tradeshop/item/ShopItemStack.java

```diff
@@ -124,17 +123,6 @@ public Map<ShopItemStackSettingKeys, ObjectHolder<?>> getItemSettings() {
         return itemSettings;
     }
 
-    public static ShopItemStack deserialize(String serialized) {
-        ShopItemStack item = new GsonProcessor().fromJson(serialized, ShopItemStack.class);
-        item.loadLegacyData();
-        item.buildMap();
-        return item;
-    }
-
-    public String serialize() {
-        return new GsonProcessor().toJson(this);
-    }
-
```

Comment:
Removing Serializable is good if it's not being used, but I see no point in removing the (de)serialize methods since at worst they just waste space. They do occasionally get used when I need to add debug to a section. In most cases when I do that they have to be removed soon after for performance like in the Hopper code(where no debug can stay since it runs so often with each hopper)